bloom_nofos/nofos/views.py

```diff
@@ -357,13 +359,42 @@ def nofo_import(request, pk=None):
 
             try:
                 nofo.filename = filename
+
+                # Always preserve metadata, but filter page breaks if checkbox is unchecked
+                preserved_metadata = preserve_subsection_metadata(nofo, sections)
                 nofo = overwrite_nofo(nofo, sections)
+
+                # If preserve_page_breaks is not checked, remove page break classes from preserved metadata
+                if request.POST.get("preserve_page_breaks") != "on":
+                    for key in preserved_metadata:
+                        if "html_class" in preserved_metadata[key]:
+                            classes = preserved_metadata[key]["html_class"].split()
+                            classes = [
+                                c for c in classes if not c.startswith("page-break")
+                            ]
```

Comment:
Maybe I am being dim, but why are we excluding "page-break*" classes?

Reply:
The idea was to copy all classes over except for the page breaks when the box is unchecked.  I was thinking other classes could have been added potentially, but in retrospect that might not actually be necessary